internal/controllers/plans.go

```diff
@@ -52,7 +52,7 @@ func (s Server) GetPlanByID(ctx echo.Context) error {
 		return model.Error(ctx, err.Error(), http.StatusInternalServerError)
 	}
 	if data.Name == "" || data.Description == "" {
-		msg := fmt.Sprintf("plan ID not found: %s", plan_id)
+		msg := fmt.Sprintf("Plan ID not found: %s", plan_id)
```

Comment:
I generally use lower case for error messages such as these. The reason is that these messages are commonly chained together and something like `Unable to defenestrate the computer: Something bad happened` look kind of strange. I'd much rather simply have it all lower case: `unable to defenestrate the computer: something bad happened`.

Of course things that are normally all upper-case are exempted, so `unable to defenestrate the computer: invalid window ID` would be preferred over `unable to defenestrate the computer: invalid window id`.